README.md

```diff
@@ -1,50 +1,40 @@
-# AKS Secure Baseline Hack
+# AKS Secure Baseline OpenHack
 
-> Welcome to the Patterns and Practices (PnP) AKS Secure Baseline (ASB) hack!
+> Welcome to the Patterns and Practices (PnP) AKS Secure Baseline (ASB) OpenHack!
 
 - The Patterns and Practices AKS Secure Baseline repo is located [here](https://github.com/mspnp/aks-secure-baseline)
-  - This repo is a summarization specifically for the hack and `should not be used for production deployments`
+  - This repo is a summarization specifically for the OpenHack and `should not be used for production deployments`
   - Please refer to the PnP repo as the `upstream repo`
 
 ## Filing Bugs
 
 > Please capture any bugs, issues or ideas on the `GitHub Board`
```

Comment:
- I created the asb-hack security group
- we will add the coaches / attendees to that group
- this seems simpler since the attendees won't know their group in advance
- AAD replication takes 10-15 minutes